src/analyses/varEq.ml

```diff
@@ -135,8 +135,11 @@ struct
       | SizeOfE _
       | SizeOfStr _
       | AlignOf _
-      | AlignOfE _ -> false
-      | UnOp (_,e,_) -> type_may_change_t e bt
+      | AlignOfE _
+      | AddrOfLabel _ -> false (* TODO: some may contain exps? *)
```

Comment:
What do you mean by the comment? These are for handling of computer gotos, so I'm fairly certain these do not occur inside Goblint.

Reply:
There's a list of cases combined here, not just `AddrOfLabel`, and some of those cases, e.g. `AlignOfE` do contain subexpressions as well, but the checks here (and in many other places) used to just shove it all under a default wildcard case, forgetting occurrence checks on some subexpressions.